package.json

```diff
@@ -50,7 +50,7 @@
     "pact:can-i-deploy": "node ./script/can-i-deploy.js",
     "pact:publish": "node ./script/publish-pacts.js",
     "pact:verify": "node ./script/verify-pacts.js",
-    "postinstall": "npm rebuild node-sass && node ./script/check-node-version.js",
+    "postinstall": "npm rebuild node-sass && node ./script/check-node-version.js && cd node_modules/web-components && npm install && npm run-script build",
```

Comment:
We are getting the `web-components` dependency from a git source, so we don't have the built version of the library. This will give us that.